components/bio-formats-plugins/src/loci/plugins/out/Exporter.java

```diff
@@ -299,14 +300,22 @@ public void run() {
             catch (DependencyException de) { }
             catch (ServiceException se) { }
 
+            OMEXMLMetadataRoot root = (OMEXMLMetadataRoot) store.getRoot();
+
             if (store == null) IJ.error("OME-XML Java library not found.");
+
+            if (root.sizeOfROIList()>0){
+                for (int roiIndex=0; roiIndex<root.sizeOfROIList(); roiIndex++) {
+                    ROI roi = root.getROI(roiIndex);
+                    root.removeROI(roi);
+                }
```

Comment:
Pull request opened against this branch to fix ROI removal: https://github.com/bramalingam/bioformats/pull/1

This seemed to fix the exported ROIs when opening ```t1.ome.tif```, removing one of the ROIs from the ROI Manager, and then exporting to a new OME-TIFF file.